hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/retransmit/PartitionOffsetChange.java

```diff
@@ -0,0 +1,23 @@
+package pl.allegro.tech.hermes.management.infrastructure.kafka.service.retransmit;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PartitionOffsetChange {
```

Comment:
we already have `PartitionOffset` object, and they don't need to be deserializable (no need for `@JsonCreator`)